java/bin/start.sh

```diff
@@ -36,13 +58,31 @@ if [ -z "$SKIP_BUILD" ]; then
   #Change PWD to root folder for Maven Build
   cd ${PROJECT_ROOT_DIR}
   mvn clean spotless:apply install -DskipTests
+  build_status=$?
+
+  if test $build_status -eq 0
+  then
+          printf "\n Code build went successful, thus we are good to go \n"
+  else
+          printf "\n We ran into some issues while buiding the jar file, seems like mvn clean install is not running fine \n"
+  fi
   mvn dependency:get -Dartifact=io.grpc:grpc-grpclb:1.40.1 -Dmaven.repo.local=./grpc_lb
 
   #Copy jar file to GCS bucket Staging folder
   echo_formatted "Copying ${PROJECT_ROOT_DIR}/target/${JAR_FILE} to  staging bucket: ${GCS_STAGING_LOCATION}/${JAR_FILE}"
   gsutil cp ${PROJECT_ROOT_DIR}/target/${JAR_FILE} ${GCS_STAGING_LOCATION}/${JAR_FILE}
+  copy1_status=$?
```

Comment:
copy1, copy2, copy3 variable names need to more appropriately named.

Reply:
Correspondingly made changes in the code.